libs/datasets/sources/can_scraper_providers.py

```diff
@@ -3,137 +3,149 @@
             unit="specimens",  # Ignores less common unit=test_encounters and unit=unique_people
-            provider="state",
+            provider=provider,
             common_field=CommonFields.TOTAL_TESTS_VIRAL,
         ),
         ccd_helpers.ScraperVariable(
             variable_name="pcr_tests_positive",
             measurement="cumulative",
             unit="specimens",  # Ignores test_encounters and unique_people
-            provider="state",
+            provider=provider,
             common_field=CommonFields.POSITIVE_TESTS_VIRAL,
         ),
         ccd_helpers.ScraperVariable(
             variable_name="total_vaccine_allocated",
             measurement="cumulative",
             unit="doses",
-            provider="state",
+            provider=provider,
             common_field=CommonFields.VACCINES_ALLOCATED,
         ),
         ccd_helpers.ScraperVariable(
             variable_name="total_vaccine_distributed",
             measurement="cumulative",
             unit="doses",
-            provider="state",
+            provider=provider,
             common_field=CommonFields.VACCINES_DISTRIBUTED,
         ),
         ccd_helpers.ScraperVariable(
             variable_name="total_vaccine_initiated",
             measurement="cumulative",
             unit="people",
-            provider="state",
+            provider=provider,
             common_field=CommonFields.VACCINATIONS_INITIATED,
         ),
         ccd_helpers.ScraperVariable(
             variable_name="total_vaccine_initiated",
             measurement="current",
             unit="percentage",
-            provider="state",
+            provider=provider,
             common_field=CommonFields.VACCINATIONS_INITIATED_PCT,
         ),
         ccd_helpers.ScraperVariable(
             variable_name="total_vaccine_completed",
             measurement="cumulative",
             unit="people",
-            provider="state",
+            provider=provider,
             common_field=CommonFields.VACCINATIONS_COMPLETED,
         ),
         ccd_helpers.ScraperVariable(
             variable_name="total_vaccine_completed",
             measurement="current",
             unit="percentage",
-            provider="state",
+            provider=provider,
             common_field=CommonFields.VACCINATIONS_COMPLETED_PCT,
         ),
         ccd_helpers.ScraperVariable(
             variable_name="total_vaccine_doses_administered",
             measurement="cumulative",
             unit="doses",
-            provider="state",
+            provider=provider,
             common_field=CommonFields.VACCINES_ADMINISTERED,
         ),
     ]
+
+
+class CANScraperStateProviders(data_source.CanScraperBase):
+    SOURCE_TYPE = "CANScrapersStateProviders"
+
+    VARIABLES = make_scraper_variables("state")
+
+
+class CANScraperCountyProviders(data_source.CanScraperBase):
+    SOURCE_TYPE = "CANScrapersCountyProviders"
+
+    VARIABLES = make_scraper_variables("county")
```

Comment:
Once upon a time I had the hope that we'll keep the list of variables here in sync with the variables available in the parquet file to reduce the chance of missed typos and accidentally dropped variables. I'd like to keep moving in that direction (in other words county and state providers have independent VARIABLES) or move towards some kind of common map from parquet file to CommonField (so the big list of all possible ScraperVariable is shared between all CanScraperBase subclasses).

Reply:
Hrm. I'd rather not use this PR as an opportunity to do a large refactor and I don't think I see the value in duplicating the variables between the county and state providers rather than re-using them via a local helper.  Is there another option you're suggesting?

Reply:
(in particular, there's no reason to treat state / county dashboards differently in terms of what variables they might have)

Reply:
By that logic I don't think there is a reason for any CanScraperBase subclass to have different variables. Add some TODO to clean this up which we won't bother doing until something breaks :-/

Reply:
The distinction is that these represent local dashboards that could provide any variables, opposed to e.g. HHSHopsitalStateDataset which we want to only contain HHS hospital data but not e.g. the HHS testing data.

I did realize that renaming can_scraper_state_providers.py to can_scraper_providers.py was probably over-generalized since I guess we consider any data coming from the parquet file as "can_scraper" data, even though most of it isn't scraped per se.  So I've renamed this to can_scraper_local_dashboard_providers.py to be a bit clearer hopefully.

Also added TODO comment per your request.